### PR TITLE
Updated image repo to Sig Network

### PR DESCRIPTION
**Pull request by auto-mausx**

### CODE REVIEW

**Comment by ppca** [comment]
https://github.com/sig-net/mpc/blob/develop/infra/multichain-mainnet/variables.tf#L20 and https://github.com/sig-net/mpc/blob/develop/infra/multichain-testnet/variables.tf#L20 probably also need to be updated?

File: infra/partner-mainnet/variables.tf [hunk 1]
```diff
@@ -17,7 +17,7 @@ variable "mig_name" {
 variable "image" {
   description = "The Docker image to deploy to GCE instances. Note: This is a public image repository used for updating your nodes, please do not change this"
   type        = string
-  default     = "us-east1-docker.pkg.dev/pagoda-discovery-platform-prod/multichain-public/multichain-mainnet:latest"
+  default     = "europe-west1-docker.pkg.dev/near-cs-mainnet/multichain-public/multichain-mainnet:latest"
```
Comment (auto-mausx) [inline infra/partner-mainnet/variables.tf:20]: @ppca

File: infra/partner-testnet/variables.tf [hunk 2]
```diff
@@ -17,7 +17,7 @@ variable "mig_name" {
 variable "image" {
   description = "The Docker image to deploy to GCE instances. Note: This is a public image repository used for updating your nodes, please do not change this"
   type        = string
-  default     = "us-east1-docker.pkg.dev/pagoda-discovery-platform-prod/multichain-public/multichain-testnet:latest"
+  default     = "europe-west1-docker.pkg.dev/near-cs-testnet/multichain-public/multichain-testnet:latest"
```
Comment (auto-mausx) [inline infra/partner-testnet/variables.tf:20]: @ppca

**Comment by auto-mausx** [comment]
> https://github.com/sig-net/mpc/blob/develop/infra/multichain-mainnet/variables.tf#L20 and https://github.com/sig-net/mpc/blob/develop/infra/multichain-testnet/variables.tf#L20 probably also need to be updated?

already changed, tagged you in above

File: infra/partner-mainnet/variables.tf (hunk 1 above)
Comment (ppca) [reply infra/partner-mainnet/variables.tf:20]: this is infra/partner-mainnet, I was referring to infra/multichain-mainnet

**Comment by auto-mausx** [comment]
Fixed, testnet image looks okay

**Review by ppca — APPROVED** [review]

**Pull request merged**